lms/templates/report_modal.html

```diff
@@ -114,7 +115,7 @@
     $("#report-form").on("ajax:success", function(event, data, status, xhr) {
         $("#report-form-wrapper").css("display", "none");
         $("#report-success-wrapper").css("display", "block");
-        $("#report-success-wrapper").focus();
+        $("#report-success-wrapper .close-modal").focus();
```

Comment:
Do we announce to screenreader users that the submission was successful? If we don't this would be confusing...